rs/execution_environment/Contributing.md

```diff
@@ -14,7 +14,7 @@ The following steps outline the recommended approach to introduce a new Manageme
 
 4. [Motoko](https://github.com/dfinity/motoko) needs to be updated. Inform *@eng-motoko* of the work required.
 
-5. Add any new types or update existing ones in `types/management_canister_types`.
+5. Add any new types or update existing ones in `types/management_canister_types`. Where adequate, stick to existing naming conventions. E.g., error variants tend to have a domain specific prefix and a clear suffix describing the error. 
```

Comment:
Could you please add an example (of an existing error variant) to illustrate your point?